reconcile/utils/slack_api.py

```diff
@@ -187,10 +210,15 @@ def __init__(
         else:
             self.config = SlackApiConfig()
 
+        # Determine the appropriate Slack API base URL based on GOV_SLACK environment variable
```

Comment:
I'm glad we're able to put this in a wrapper to avoid call this billions of times. 👍